demos/src/demo.scss

```diff
@@ -2,21 +2,112 @@ $o-typography-is-silent: false;
 
 @import '../../main';
 
-.demo-row {
-	clear: both;
+html {
+	@include oColorsFor(page);
+	overflow-y: scroll;
+
+	// type defaults
+	font-family: $o-typography-sans;
+	-ms-text-size-adjust: 100%;
+	-webkit-text-size-adjust: 100%;
+	text-rendering: optimizeSpeed;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
+
+.demo {
+	// prevent images and media from expanding its container
+	img,
+	svg,
+	video,
+	object,
+	iframe,
+	figure,
+	figure img,
+	figure svg {
+		max-width: 100%;
+		height: auto;
+		display: block;
+	}
+	figure {
+		margin-right: 0;
+		margin-left: 0;
+	}
+
+	a {
+		@include oColorsFor(link);
+	}
+}
+
+.demo-article {
+	margin: 10px 0;
+
+	&__body {
```

Comment:
if you take it outside as `.demo-article__body` it's less specific, and with bem, we don't really need to worry about that